vault/src/system.rs

```diff
@@ -69,6 +69,11 @@ pub struct VaultServiceConfig {
     #[clap(long)]
     pub no_issue_execution: bool,
 
+    /// Don't try to execute **all** redeem and replace requests,
+    /// the Vault will still submit proofs for their own
+    #[clap(long)]
+    pub no_execute_all: bool,
```

Comment:
Maybe this should be opt-in rather than opt-out? It will make rescanning on startup take a lot longer